### PR TITLE
Change language for error message in donation for different language

### PR DESCRIPTION
**Pull request by rishsingh07**
Passed locale in getStripe

### CODE REVIEW

**Comment by sagararyal** [comment]
I'm not sure this solves. Also are stripe's locale and our locale 1:1?

File: src/utils/stripe/getStripe.tsx [hunk 1]
```diff
@@ -3,9 +3,10 @@ import { Stripe, loadStripe } from '@stripe/stripe-js';
 let stripePromise: Promise<Stripe | null>;
 
 const getStripe = (paymentSetup:any) => {
+  let lang = localStorage.getItem('language') || 'en';
   const key = paymentSetup?.gateways?.stripe?.authorization.stripePublishableKey ? paymentSetup?.gateways?.stripe?.authorization.stripePublishableKey : paymentSetup?.gateways?.stripe?.stripePublishableKey;
   const account = paymentSetup?.gateways?.stripe?.authorization.accountId;
-  stripePromise = loadStripe(key,{stripeAccount:account});
+  stripePromise = loadStripe(key,{stripeAccount:account,locale:lang});
```
Comment (norbertschuler) [inline src/utils/stripe/getStripe.tsx:9]: I tested this before and I saw no difference from not doing it. Stripe is already loading some translation resources based by the browser language and some translations (which maybe are only for development purpose) do not seem to get translated:

Bildschirmfoto 2021-03-03 um 10 51 53Bildschirmfoto 2021-03-03 um 10 40 59

But for the case that someone wants to use our website in another language as the default language of his browser, we should keep this patch anyway.

**Comment by sagararyal** [comment]
@norbertschuler if good to merge, you can do so.

**Comment by norbertschuler** [comment]
@sagararyal @rishsingh07 we first need to find out why the donation test failed. (And fix linting errors. And mark the PR as ready to review 😁)

**Review by harshvitra — APPROVED** [review]

**Pull request merged**